homeworks/iryna.gychka_irina-hychka/homework_5/task_1.js

```diff
@@ -0,0 +1,81 @@
+// Task 1.1 Converts a numeric array to array of sums
+const xArray = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+const yArray = [0, 0, 0, 0, 0];
+const zArray = [10, -10, 10, -10, 10];
+
+function sumArray(n) {
+    const sum = [];
+    for (let el = 1; el <= n.length; el++) {
+        sum.push(n.slice(0, el).reduce((res, sub) => (res + sub)));
+    }
+    return sum;
+}
```

Comment:
push... slice... reduce... to many operations, if I wanted to make array out of array I would think about `map()` first